core/src/main/java/tc/oc/pgm/flag/state/Respawning.java

```diff
@@ -55,22 +55,31 @@ public void enterState() {
     if (!Duration.ZERO.equals(respawnTime)) {
       // Respawn is delayed
 
-      String displayPostName = "";
+      // String displayPostName = "";
       String postName = this.respawnToPost.getPostName();
       if (postName != null) {
-        displayPostName = " to " + net.md_5.bungee.api.ChatColor.AQUA + postName;
+        // displayPostName = " to " + net.md_5.bungee.api.ChatColor.AQUA + postName;
+        this.flag
+            .getMatch()
+            .sendMessage(
+                new PersonalizedTranslatable(
+                    "flag.willRespawnNamed",
+                    this.flag.getComponentName(),
+                    net.md_5.bungee.api.ChatColor.AQUA + postName,
+                    new PersonalizedText(
+                        PeriodFormats.briefNaturalApproximate(respawnTime),
+                        net.md_5.bungee.api.ChatColor.AQUA)));
+      } else {
+        this.flag
+            .getMatch()
+            .sendMessage(
+                new PersonalizedTranslatable(
+                    "flag.willRespawn",
+                    this.flag.getComponentName(),
+                    new PersonalizedText(
+                        PeriodFormats.briefNaturalApproximate(respawnTime),
+                        net.md_5.bungee.api.ChatColor.AQUA)));
```

Comment:
Use `TranslatableComponent.of()` instead of `PersonalizedTranslatable` (Should've told you that earlier sorry